docs/tests/getting-started/github-actions.yml

```diff
@@ -24,15 +20,15 @@ jobs:
         uses: actions/checkout@v2
 
       # You need to run `dagger project init` locally before and commit the cue.mod directory to the repository with its contents
-      - name: Project update
-        uses: dagger/dagger-for-github@v2
-        with:
-          version: 0.2
-          # To pin external dependencies, you can use `project update github.com/[package-source]@v[n]`
-          args: project update
-
       - name: Deploy to Netlify
         uses: dagger/dagger-for-github@v2
         # See all options at https://github.com/dagger/dagger-for-github
         with:
-          args: do deploy
+          version: 0.2
+          # To pin external dependencies, you can use `project update github.com/[package-source]@v[n]`
+          cmds: |
+            project update
+            do deploy
```

Comment:
I really like this pattern.

What happens when `project update` fails? Before this, the step would make it clear why it failed, but now I don't know.

Reply:
If `project update` fails then it will exit immediately and don't process the next command

Reply:
Here is what it looks like with collapsible groups to split logs per cmd: https://github.com/dagger/dagger-for-github/runs/6022757575?check_suite_focus=true#step:3:12

Reply:
That sounds good! How does it look when `project update` fails?

Reply:
![image](https://user-images.githubusercontent.com/1951866/163403710-de2c59f3-4c91-4e5f-ada0-9d890678c921.png)